corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -203,4 +199,28 @@ public int dropTable(String namespace, String tablename) {
         log.info("\n======================\n");
         return tableSize;
     }
+
+    /**
+     * Loads the table with random data
+     * @param namespace - the namespace where the table belongs
+     * @param tablename - table name without the namespace
+     * @return - number of entries in the table
+     */
+    public int loadTable(String namespace, String tablename) {
+        CorfuTable<byte[], byte[]>
+                corfuTable = runtime.getObjectsView().build()
+                .setTypeToken(new TypeToken<CorfuTable<byte[], byte[]>>() {})
+                .setStreamName("test")
+                .open();
+        final int batchSize = 1000;
+        final int numItems = 1000;
+        byte[] bigBadBuf = "JUST_A_RANDOMLY_LOOOOOOOOOOOOOOOOOOOOONG_STRING".getBytes();
+        for (int i = numItems; i> 0; i--) {
+            log.info("loadTable: Batches left {}", i);
+            for (int j = 0; j < batchSize; j++) {
+                corfuTable.put(bigBadBuf, bigBadBuf);
```

Comment:
this will write the same entry over and over again.  Is it the intention?

Reply:
yes - it was meant to test checkpointing where we want to see if the checkpointer reduces the space. space will only be reduced if there is only one real key-value pair